lib/tasks/migrations/migrations.rake

```diff
@@ -74,6 +74,44 @@ namespace :migrations do
     exit has_encountred_issue
   end
 
+  task :rooms, [:start, :stop] => :environment do |_task, args|
+    start, stop = range(args)
+    has_encountred_issue = 0
+
+    Room.select(:uid, :name, :bbb_id, :last_session, :user_id)
```

Comment:
It's always recommended to include the `:id` attribute (or the configured primary key attribute).
However, when doing batch processing it's required to provide the `:id` attribute since batching will occure by ordering, limitting and offseting records based on their ids.
This will throw an exception,

Reply:
I suspect that this is subject to `N+1` problem along with the users task,
We can optimize them in a new PR,